docs/docsite/rst/porting_guides/porting_guide_2.10.rst

```diff
@@ -282,23 +282,23 @@ community.general
   for example replace ``community.general.hetzner_firewall_info`` in a task by ``community.hrobot.firewall_info``.
 
   If you use ansible-base and installed ``community.general`` manually and rely on the Hetzner Robot modules, you have to make sure to install the ``community.hrobot`` collection as well.
-  If you are using FQCNs, i.e. ``community.general.hetzner_failover_ip`` instead of ``hetzner_failover_ip``, it will continue working, but we still recommend to adjust the FQCNs as well.
+  If you are using FQCNs, that is. ``community.general.hetzner_failover_ip`` instead of ``hetzner_failover_ip``, it will continue working, but we still recommend to adjust the FQCNs as well.
```

Comment:
You kept the period after `i.e`, that must also go:
```suggestion
  If you are using FQCNs, that is ``community.general.hetzner_failover_ip`` instead of ``hetzner_failover_ip``, it will continue working, but we still recommend to adjust the FQCNs as well.
```
Same in multiple places below.